dfndb/models.py

```diff
@@ -28,10 +28,9 @@ def __str__(self):
 
     def clean(self):
         """
-        Use the formula to calculate the mass if not specified.
+        Use the formula to calculate the mass.
         """
-        if self.mass == 0:
-            self.mass = Formula(self.formula).mass
+        self.mass = Formula(self.formula).mass
```

Comment:
I wonder if this `mass` field should be converted into a property now? I noticed in the tests for this model that the mass is set on creation of the object in one of them (`TestCompound.test_unique_together`). We probably want to prevent that being possible now.

Maybe we remove the `clean` method entirely and replace it with a property for `mass`, @dandavies99 ?

Reply:
Yes it's probably simpler just to move that logic into a property. There are a few other examples of where `@property` is used in that file for other models @CWestICL.